.github/workflows/ui-test.yml

```diff
@@ -0,0 +1,69 @@
+name: Automated Tests - PR
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  workflow_dispatch:
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - uses: actions/checkout@v2
+
+      # Runs a single command using the runners shell to change apiEndPoint.ts content to 
+      # change env from prod to stage
+      - name: print content of apiendpoint.ts file in Linux - Before
+        if: ${{ runner.os == 'Linux'}}
+        run: cat /home/runner/work/fabric8-analytics-vscode-extension/fabric8-analytics-vscode-extension/src/apiendpoint.ts
+
+      - name: Change env from Prod to Stage for Linux
+        if: ${{ runner.os == 'Linux'}}
+        run: sed -i 's/connect-api-2445582058137.production/analytics-preview-2445582058137.staging/g' /home/runner/work/fabric8-analytics-vscode-extension/fabric8-analytics-vscode-extension/src/apiendpoint.ts && sed -i 's/ad467b765e5c8a8a5ca745a1f32b8487/3e42fa66f65124e6b1266a23431e3d08/g' /home/runner/work/fabric8-analytics-vscode-extension/fabric8-analytics-vscode-extension/src/apiendpoint.ts
+
+      - name: print content of apiendpoint.ts file in Linux - After
+        if: ${{ runner.os == 'Linux'}}
+        run: cat /home/runner/work/fabric8-analytics-vscode-extension/fabric8-analytics-vscode-extension/src/apiendpoint.ts
+
+
+      - name: print content of apiendpoint.ts file in Windows - Before
+        if: ${{ runner.os != 'Linux'}}
+        run: more D:\a\fabric8-analytics-vscode-extension\fabric8-analytics-vscode-extension\src\apiendpoint.ts
+
+      - name: Change env from Prod to Stage for Windows-1
+        if: ${{ runner.os != 'Linux'}}
```

Comment:
why only for linux ?

Reply:
it's for both. I have to add differently for both linux and windows as file path works differently in these OS's

Reply:
It's for Windows (as it says os != 'Linux')
refer to line no 28 for Linux